app/src/main/java/com/vultisig/wallet/ui/screens/ChainCoinScreen.kt

```diff
@@ -120,39 +128,41 @@ fun ChainCoinScreen(navController: NavHostController) {
-                    minHeight = dimens.minHeightButton,
-                    backgroundColor = appColor.turquoise800,
-                    textColor = appColor.oxfordBlue800,
-                    iconColor = appColor.turquoise800,
-                    textStyle = Theme.montserrat.titleLarge,
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(
-                            start = dimens.marginMedium,
-                            end = dimens.marginMedium,
-                            bottom = dimens.marginMedium,
-                        ),
-                    centerContent = {
-                        Row(
-                            horizontalArrangement = Arrangement.Center,
-                            verticalAlignment = Alignment.CenterVertically
-                        ) {
-                            Icon(
-                                painter = painterResource(id = R.drawable.logo_button),
-                                contentDescription = null,
-                                modifier = Modifier.width(dimens.medium1),
-                                tint = MaterialTheme.colorScheme.onPrimary
-                            )
-                            Text(
-                                text = "BUY \$VTX",
-                                color = MaterialTheme.colorScheme.onPrimary,
-                                style = Theme.montserrat.titleLarge
-                            )
+                if (buyVltiButtonVisible) {
+                    MultiColorButton(
+                        minHeight = dimens.minHeightButton,
+                        backgroundColor = appColor.turquoise800,
+                        textColor = appColor.oxfordBlue800,
+                        iconColor = appColor.turquoise800,
+                        textStyle = Theme.montserrat.titleLarge,
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(
+                                start = dimens.marginMedium,
+                                end = dimens.marginMedium,
+                                bottom = dimens.marginMedium,
+                            ),
+                        centerContent = {
+                            Row(
+                                horizontalArrangement = Arrangement.Center,
+                                verticalAlignment = Alignment.CenterVertically
+                            ) {
+                                Icon(
+                                    painter = painterResource(id = R.drawable.logo_button),
+                                    contentDescription = null,
+                                    modifier = Modifier.width(dimens.medium1),
+                                    tint = MaterialTheme.colorScheme.onPrimary
+                                )
+                                Text(
+                                    text = "BUY \$VTX",
```

Comment:
This can be moved to strings.xml for easier i18n later
```suggestion
                                    text = stringResource(R.string.chain_accounts_buy_vtx)
```